extensions/ql-vscode/src/model-editor/methods-usage/methods-usage-data-provider.ts

```diff
@@ -165,7 +165,9 @@ export class MethodsUsageDataProvider
 
 export type MethodsUsageTreeViewItem = Method | Usage;
 
-function isExternalApiUsage(item: MethodsUsageTreeViewItem): item is Method {
+export function isExternalApiMethod(
```

Comment:
We should consider putting this in a shared file for the model-editor.

Reply:
Agree. Shall we have a separate PR to move it to `method.ts` and rename it? I'd call it `isMethod` as we're gradually moving away from the term External API.

Reply:
Sounds good!